src/game/Spells/SpellAuras.cpp

```diff
@@ -6234,7 +6234,14 @@ void Aura::PeriodicDummyTick()
                 }
                 case 8067:                                  // Party Time!
                 {
-                    target->HandleEmoteCommand(PickRandomValue(EMOTE_ONESHOT_APPLAUD, EMOTE_ONESHOT_CHEER, EMOTE_ONESHOT_CHICKEN, EMOTE_ONESHOT_LAUGH, EMOTE_ONESHOT_DANCE));
+                    if (!target->HasAura(8067))
```

Comment:
This check will never be true, since you are in the handler of spell 8067.

Reply:
I've removed this redundant check.